web_app/utils/oidc_tools.py

```diff
@@ -30,6 +31,7 @@ def _cached_get_info(oidc_csrf_token: str) -> UserInfo:
         # pylint:disable=unused-argument
         logging.warning(f"Cache Miss: CurrentUser._cached_get_info({oidc_csrf_token=})")
         resp: Dict[str, Any] = oidc.user_getinfo(["preferred_username", "groups"])
+        resp["access_token"] = oidc.get_access_token()
```

Comment:
FYI: the access token is only valid for 5 minutes, so you don't want to cache this for 60 minutes.

Reply:
Admittedly, I'm not sure how flask treats all this stuff with multiple sessions, hence the keying on `oidc_csrf_token`.

I've decreased the cache timeout to 4:59